app/assets/stylesheets/thredded/components/_messageboard.scss

```diff
@@ -43,6 +43,15 @@
   margin-bottom: $thredded-small-spacing / 2;
 }
 
+.threadded--messageboard--icon {
+  fill: currentColor;
```

Comment:
Please replace this line with

```scss
@extend %thredded--icon;
```

This makes it easier for the user to customize all the icons.